support/build/config-submenu.sh

```diff
@@ -92,6 +104,39 @@ else
 	exec 7>&1
 fi
 
+#
+# GENERATE LIST OF FINAL LIBRARY PATHS
+#
+LIBS=()
+for ARG_PATH in "${ARG_PATHS[@]}" "$@"; do
+	# Remove trailing slashes
+	ARG_PATH="${ARG_PATH%/}"
+
+	# Does Config.uk exist?
+	CONFIG_UK="${ARG_PATH}/Config.uk"
+	if [ ! -f "${CONFIG_UK}" ]; then
+		if [ $OPT_QUIET -ne 0 ]; then
+			echo "Could not find \"Config.uk\" under \"${ARG_PATH}\". Skipping..." 1>&2
+		fi
+		continue
+	fi
+
+	DONT_INCLUDE=1
+	for ARG_EXCLSUFFIX in "${ARG_EXCLSUFFIXES[@]}"; do
+		if [[ "${ARG_PATH}" == *"${ARG_EXCLSUFFIX}" ]]; then
```

Comment:
Because the `EXCLSUFFIX` is only a matching expression on the right side of the path, it cannot be expanded to a full path. However, since the build system works with fully qualified paths, I can make sure that a path is always matched in its FQ form within `config-submenu`. So far, I did not see relative paths because `config-submenu` is feeded by the build system but with such a change we are then 100% sure that we achieve the same behavior between Makefile and menu.

Reply:
> Because the EXCLSUFFIX is only a matching expression on the right side of the path, it cannot be expanded to a full path.

Similarly to https://github.com/unikraft/unikraft/pull/1104#discussion_r1334111702 can you please provide an example? What I understand from it is that you match `*lib/uksched`, so `$(realpath lib/uksched)` would resolve into `/home/user/unikraft/lib/uksched`. Clearly I'm missing something? 🤔

Reply:
This is correct. You could even do something weird like `*ib/uksched` which would still match with `/home/user/unikraft/lib/uksched`. Everything that matches is filtered out => skipped by the build system.